scripts/build_addons.sh

```diff
@@ -0,0 +1,55 @@
+# !/bin/bash
+
+if [ "$#" -eq 0 ]; then
+	echo "usage build_addons <product path> <release/debug - optional>"
```

Comment:
ffmpeg base path missing?
build path missing (optional)

product path should never be passed